packages/core/bootstrap/src/lib/middleware/cache-warmer/config.ts

```diff
@@ -35,7 +35,7 @@ export interface Config {
 }
 
 export function get(): Config {
-  const warmupInterval = Number(process.env.WARMUP_INTERVAL)
+  const warmupInterval = Number(getEnv('WARMUP_INTERVAL'))
```

Comment:
So ones like this because they don't have the context passed in won't be able to have an EA coded default override.. could we write that as a comment somewhere, like in the JSdoc for `getEnv` maybe?

I"ll also add it to the project ideas board to refactor for all being default config overridden.

Reply:
Yeah good idea to add it to the board. I added more detail to the `getEnv` so it should be clear that `context` needs to be passed in.